code/src/index.css

```diff
@@ -5,9 +5,277 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+  box-sizing: border-box;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.loading-overlay {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: black;
+}
+
+.loading-container {
+  width: 200px;
+  border: 3px solid black;
+}
+
+.loading {
+  height: 10px;
+  width: 200px;
+  background: red;
+  animation: loader infinite 0.6s;
+}
+
+@keyframes loader {
+  0% {
+    width: 0%;
+  }
+
+  50% {
+    width: 50%;
+  }
+
+  100% {
+    width: 100%;
+  }
+}
+
+.outer-container {
+  display: flex;
+  flex-direction: column;
+  margin: 0 10px 0 10px;
+}
+
+.movie-container {
+  display: flex;
+  flex-direction: column;
+  margin: 20px 0;
+  background-color: black;
+  border: 2px solid;
+}
+
+.movie-container:hover {
+  animation: frame infinite 0.8s;
+}
+
+@keyframes frame {
+  0% {
+    border-color: white;
+  }
+
+  50% {
+    border-color: #ff4d4d;
+  }
+
+  75% {
+    border-color: #e60000;
+  }
+
+  100% {
+    border-color: #800000;
+  }
+}
+
+.overlay-container {
+  width: 100%;
+  position: relative;
+}
+
+.overlay {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: flex-start;
+}
+
+.main-movie {
+  width: 100%;
+  display: block;
+}
+
+.movie-anchor {
+  display: none;
+  color: white;
+  font-size: 32px;
+  font-weight: 700;
+  padding: 15px;
+  text-decoration: none;
+}
+
+.overlay:hover {
+  background-color: rgba(46, 49, 49, 0.7);
+}
+
+.overlay:hover > .movie-anchor,
+.overlay:hover > .release {
+  display: block;
+}
+
+.release {
+  display: none;
+  color: white;
+  font-size: 18px;
+  padding: 15px;
+  margin: 0;
+}
+
+.background {
+  display: flex;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  flex-direction: column;
+  justify-content: flex-start;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  height: 100vh;
+}
+
+.summary {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 50px;
+  font-size: 16px;
+  font-weight: 700;
+  color: white;
+}
+
+.movie-detail {
+  width: 180px;
+  border: 5px solid#fff;
+}
+
+.detail {
+  display: flex;
+  flex-direction: column;
+  margin-top: 25px;
+}
+
+.title {
+  font-size: 24px;
+}
+
+.rating {
+  color: red;
+  font-size: 17px;
+  font-weight: 600;
+  margin-top: 5px;
+}
+
+.overview {
```

Comment:
Nice that you have a darker background for the text! It definitely improves readability.